docs/requirements/requirements.md

```diff
@@ -0,0 +1,112 @@
+# Hardware requirements of TwPM module
+
+[tcg_client_v13r27]:<https://trustedcomputinggroup.org/wp-content/uploads/TCG_PCClientTPMInterfaceSpecification_TIS__1-3_27_03212013.pdf>
+[tcg_client_PTP_v1p5r14]:<https://trustedcomputinggroup.org/wp-content/uploads/PC-Client-Specific-Platform-TPM-Profile-for-TPM-2p0-v1p05p_r14_pub.pdf>
+
+## Acronyms
+
+Legend:
+
+* PTP – Platform TPM Profile
+* CRB – Command Response Buffer interface
+* DDWG – Device Driver’s Writers Guide
+* Certification PP – Certification Protection Profile
+* TIS – TPM Interface Specification
+
+## Requirements
+
+### Power supply
+
+#### Power supply voltage
+
+* TPM MUST support a supply voltage of 1.8V or 3.3V.
+* TPM MAY support supply and I/O voltages of both 1.8V and 3.3V.
+* TPM MAY support other supply voltages.
+
+Source: [tcg_client_v13r27] l.1373-1375
```

Comment:
Non-uniform style, some references have full word (`line`) others only `l.`.

Reply:
(corrected in next commit)